### PR TITLE
Add the instruction on how to upgrade v0.10 and v0.11 to v0.12

### PR DESCRIPTION
**Pull request by houshengbo**
Fixes #

## Proposed Changes

*
*
*

**Release Note**

```release-note

```

### CODE REVIEW

File: doc/upgrade_guide.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+## v0.10.0, v0.11.x -> v0.12.0
+
+Both of v0.10 and v0.11 versions are able to upgrade to the version v0.12.0 by running the following command:
+
+```
+kubectl apply -f https://github.com/knative/serving-operator/releases/download/v0.12.0/serving-operator.yaml
+```
+
+## v0.9.0 or earlier
```
Comment (jcrossley3) [inline doc/upgrade_guide.md]: Can we please just delete our 0.9.0 release and never speak of it again? It's called 0.9.0 yet it installs 0.8.0. It was a mistake.

Comment (k4leung4) [reply doc/upgrade_guide.md]: +1

Comment (houshengbo) [reply doc/upgrade_guide.md]: OK

**Pull request merged**